src/test/java/com/google/sps/GetCookieGivenNameTest.java

```diff
@@ -0,0 +1,118 @@
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import java.io.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.sps.CookieUtils; 
+
+@RunWith(JUnit4.class)
+public final class GetCookieGivenNameTest {
+    private CookieUtils cookieUtils;
+
+  @Before
+  public void setUp() {
+    cookieUtils = new CookieUtils();
```

Comment:
We actually might want to have a separate utility function declared in each test method. It's usually best to avoid sharing objects across multiple tests.